pkg/skaffold/event/event.go

```diff
@@ -677,6 +730,14 @@ func ResetStateOnBuild() {
 	handler.setState(newState)
 }
 
+// ResetStateOnTest resets the deploy, sync and status check state
+func ResetStateOnTest() {
+	newState := handler.getState()
+	newState.DeployState.Status = NotStarted
```

Comment:
shouldn't this reset only `TestState`?

Reply:
also I couldn't find a reference for `ResetStateOnTest`. Should it be called in `doDev` function in `dev.go`?

Reply:
Updated. Thanks for catching.